CONTRIBUTING.md

```diff
@@ -89,17 +89,76 @@ to make reviews and retrospection easy. Use your git commits to provide context
 for the reviewers, and the folks who will be reading the codebase in the months
 and years to come.
 
-Finalized commit messages should look similar to the following format:
+Kanister recommends using [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/) format:
 
 ```text
-Short one line title
+<type>[optional scope]: <description>
 
-An explanation of the problem, providing context, and why the change is being
-made.
+[optional body]
```

Comment:
Hi @hairyhum ,
Sorry, if I am mistaken. But I thought we discussed that this format should be followed in PR titles and desc.
What I had in mind for commits was a rule set like this


Commits should be of format

```
Title
Description
```

- It's totally fine if you are able to convey the change in title itself, you don't necessarily have to provide desc.
- Commit titles should not start with lower case. So `add go mod check while init` -> `Add go mod check while init`, same applies to PR titles as well
- Title should imperative so use `Add go lint` instead of `Added go lint`, same applies to PR titles as well
- Do not end title with period
- Desc (if present) should specify why did you make that change
- Don't assume reviewer understand the problem

This is very rough, but I think for commits we should have this kind of rules and for PR titles we can have the type and scope etc.

Reply:
So your proposal is to have PR title/description convention **instead of** commit convention?